addons/project/static/src/views/project_task_kanban/project_task_kanban_compiler.js

```diff
@@ -0,0 +1,50 @@
+/** @odoo-module **/
+
+import { KanbanCompiler } from "@web/views/kanban/kanban_compiler";
+import { append, createElement } from "@web/core/utils/xml";
+
+export class ProjectTaskKanbanCompiler extends KanbanCompiler {
+    setup() {
+        super.setup();
+        this.compilers.push(
+            { selector: ".subtask_list_button", fn: this.compileSListButton },
+            { selector: "div.kanban_bottom_subtasks_section", fn: this.compileSListComponent },
+        );
+    }
+
+    /**
+     * @param {Element} el
+     * @returns {Element}
+     */
+    compileSListButton(el) {
+        el.setAttribute("t-on-click", `() => state.folded = !state.folded`);
```

Comment:
I am wondering if we don't have to use the `__comp__` here. 🤔 
```suggestion
        el.setAttribute("t-on-click", `() => __comp__.state.folded = !__comp__.state.folded`);
```

Maybe, for the moment, it works because the state is in the renderingContext. 🤔

Reply:
I tried with and without it and it did not make a difference, but the state had to be in the `renderingContext`. Removing it would lead to an error.

Reply:
Oh 😢 
Thanks for the testing. 🙂